security/pkg/nodeagent/cache/mock/secretcache_mock.go

```diff
@@ -49,7 +49,7 @@ func NewMockCAClient(mockCertChain1st, mockCertChainRemain []string, failureRate
 	return &cl
 }
 
-func (c *CAClient) CSRSign(ctx context.Context, csrPEM []byte, exchangedToken string,
+func (c *CAClient) CSRSign(ctx context.Context, reqID string, csrPEM []byte, exchangedToken string,
 	certValidTTLInSec int64) ([]string /*PEM-encoded certificate chain*/, error) {
```

Comment:
We could use a struct to wrap some parameters, but I'm okay with current function parameter list.

Reply:
Sure. I think it's OK for now.